pkg/test/framework/components/echo/annotations.go

```diff
@@ -36,17 +36,18 @@ type Annotation struct {
 }
 
 var (
-	SidecarInject                  = workloadAnnotation(annotation.SidecarInject.Name, "true")
-	SidecarRewriteAppHTTPProbers   = workloadAnnotation(annotation.SidecarRewriteAppHTTPProbers.Name, "")
-	SidecarBootstrapOverride       = workloadAnnotation(annotation.SidecarBootstrapOverride.Name, "")
-	SidecarVolumeMount             = workloadAnnotation(annotation.SidecarUserVolumeMount.Name, "")
-	SidecarVolume                  = workloadAnnotation(annotation.SidecarUserVolume.Name, "")
-	SidecarConfig                  = workloadAnnotation(annotation.ProxyConfig.Name, "")
-	SidecarInterceptionMode        = workloadAnnotation(annotation.SidecarInterceptionMode.Name, "REDIRECT")
-	SidecarIncludeInboundPorts     = workloadAnnotation(annotation.SidecarTrafficIncludeInboundPorts.Name, "")
-	SidecarIncludeOutboundIPRanges = workloadAnnotation(annotation.SidecarTrafficIncludeOutboundIPRanges.Name, "")
-	SidecarProxyConfig             = workloadAnnotation(annotation.ProxyConfig.Name, "")
-	SidecarInjectTemplates         = workloadAnnotation(inject.TemplatesAnnotation, "")
+	SidecarInject                       = workloadAnnotation(annotation.SidecarInject.Name, "true")
+	SidecarRewriteAppHTTPProbers        = workloadAnnotation(annotation.SidecarRewriteAppHTTPProbers.Name, "")
+	SidecarBootstrapOverride            = workloadAnnotation(annotation.SidecarBootstrapOverride.Name, "")
+	SidecarVolumeMount                  = workloadAnnotation(annotation.SidecarUserVolumeMount.Name, "")
+	SidecarVolume                       = workloadAnnotation(annotation.SidecarUserVolume.Name, "")
+	SidecarConfig                       = workloadAnnotation(annotation.ProxyConfig.Name, "")
+	SidecarInterceptionMode             = workloadAnnotation(annotation.SidecarInterceptionMode.Name, "REDIRECT")
+	SidecarIncludeInboundPorts          = workloadAnnotation(annotation.SidecarTrafficIncludeInboundPorts.Name, "")
+	SidecarIncludeOutboundIPRanges      = workloadAnnotation(annotation.SidecarTrafficIncludeOutboundIPRanges.Name, "")
+	SidecarProxyConfig                  = workloadAnnotation(annotation.ProxyConfig.Name, "")
+	SidecarInjectTemplates              = workloadAnnotation(inject.TemplatesAnnotation, "")
+	WorkloadEnableAutoMtlsWithPeerAuthn = workloadAnnotation("security.istio.io/autoMtlsWithPeerAuthn", "true")
```

Comment:
what is this for?

Reply:
removed. i originally want to add support for per workload config option,but later find that might be not necessary and cause some trouble instead.